plugins/source/launchdarkly/resources/services/auditlog/auditlog_entries.go

```diff
@@ -0,0 +1,87 @@
+package auditlog
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/cloudquery/cloudquery/plugins/source/launchdarkly/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	ldapi "github.com/launchdarkly/api-client-go/v11"
+)
+
+func AuditLogEntries() *schema.Table {
+	return &schema.Table{
+		Name:        "launchdarkly_auditlog_entries",
+		Description: `https://apidocs.launchdarkly.com/tag/Audit-log#operation/getAuditLogEntries`,
+		Resolver:    fetchAuditLogEntries,
+		Transform:   transformers.TransformWithStruct(&ldapi.AuditLogEntryListingRep{}, transformers.WithSkipFields("Id", "Date"), transformers.WithSkipFields("Links")),
+		Columns: schema.ColumnList{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("Id"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "date",
+				Type:     schema.TypeTimestamp,
+				Resolver: client.UnixTimeResolver("Date"),
+				CreationOptions: schema.ColumnCreationOptions{
+					IncrementalKey: true,
+				},
+			},
+		},
+		IsIncremental: true,
+	}
+}
+
+const key = "auditlog_entries"
+
+func fetchAuditLogEntries(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	cl := meta.(*client.Client)
+
+	cursor := int64(0)
+	if cl.Backend != nil {
+		value, err := cl.Backend.Get(ctx, key, cl.ID())
+		if err != nil {
+			return fmt.Errorf("failed to retrieve state from backend: %w", err)
+		}
+		if value != "" {
+			valInt, err := strconv.ParseInt(value, 10, 64)
+			if err != nil {
+				return fmt.Errorf("retrieved invalid state value: %q %w", value, err)
+			}
+			cursor = valInt
+		}
+	}
+
+	const limit = 20
+	for {
+		list, b, err := cl.Services.AuditLogApi.GetAuditLogEntries(ctx).Limit(limit).After(cursor).Execute()
+		if err != nil {
+			return err
+		}
+		b.Body.Close()
+		res <- list.Items
+
+		if l := len(list.Items); l > 0 {
+			cursor = list.Items[l-1].Date
```

Comment:
What order are the log entries returned in? I'm a bit confused about why we're iterating over the items in reverse order here 🤔

Reply:
We're getting the last item from the list and setting the cursor to it, not iterating. The order is not documented and unlike other endpoints can't be specified.

Reply:
Removed that ambiguity with [`764b355` (#7191)](https://github.com/cloudquery/cloudquery/pull/7191/commits/764b35513263b63714ead19cc0f8f6c280ac8ff7) and we now compare the cursor with each item. Also started storing it once per page if changed with [`6831918` (#7191)](https://github.com/cloudquery/cloudquery/pull/7191/commits/6831918b7cf65161581249c4cdfff6c67bee6cad).